usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -3424,8 +3424,8 @@ arc_reclaim_thread(void)
 			 * even if we aren't being signalled)
 			 */
 			CALLB_CPR_SAFE_BEGIN(&cpr);
-			(void) cv_timedwait(&arc_reclaim_thread_cv,
-			    &arc_reclaim_lock, ddi_get_lbolt() + hz);
+			(void) cv_timedwait_hires(&arc_reclaim_thread_cv,
+			    &arc_reclaim_lock, SEC2NSEC(1), MSEC2NSEC(1), 0);
```

Comment:
Previously, you were waking up every 1s but with 10ms granularity (assuming `hires_tick` is not tweaked).  Now, you are using 1ms granularity.  While I don't think that should make a difference, it is different behavior none the less.  Have you noticed any actual difference?